src/App.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react'
+import SplitPane from 'react-split-pane'
+
+import Overlay from './components/preview'
+import EditorPanel from './components/settingsMenu'
+// import onClike from './components/generate'
+
+
+import { SettingsContext } from './lib/contexts'
+import { loadCss, loadStorage } from './lib/utils'
+
+import './App.css'
+
+import {
+  GURMUKHI,
+  TRANSLITERATION_ENGLISH,
+  TRANSLITERATION_DEVNAGRI,
+  TRANSLITERATION_URDU,
+  TRANSLATION_ENGLISH,
+  TRANSLATION_PUNJABI,
+  TRANSLATION_SPANSISH,
+} from './lib/mool-mantar'
+
+loadStorage()
```

Comment:
Do these calls inside app.

```
useEffect(() => {
  loadStorage()
  loadCss()
})
```

It is clearer that it is done once, on App mount then